examples/ColorSpaceInterpolations.elm

```diff
@@ -0,0 +1,231 @@
+module ColorSpaceInterpolations exposing (main)
+
+{-| This module shows how to build some simple colour space palettes.
+-}
+
+import Browser
+import Color exposing (Color, rgb255)
+import Hex
+import Html exposing (Html, div)
+import Html.Attributes exposing (class, for, href, id, selected, style, type_, value)
+import Html.Events exposing (onInput)
+import Interpolation exposing (Interpolator, hcl, hclLong, hsl, hslLong, lab, rgb)
```

Comment:
I would prefer if (for didactical reasons) we could keep these functions prefixed. It makes it easier for readers to understand where these functions come from. (The type alias is fine)